### PR TITLE
Improved copy for 'Can't use the map...'

### PR DESCRIPTION
**Pull request by lucascumsille**
Fixes: https://github.com/mysociety/societyworks/issues/4483

This new copy for 'Can't use the map...' should provide more context to users with visual impairments.

Current copy:'Can’t use the map? [Skip this step.](https://fixmystreet.cheshireeast.gov.uk/report/new?latitude=53.145324&skipped=1&pc=CW11+1HZ&longitude=-2.370437)' 
Changed to 'Can’t use the map to start a report?
[Skip this step →](https://cheshireeast.fixmystreet.localhost:3000/report/new?pc=&longitude=-1.89801&latitude=52.47766&skipped=1)'

Screenshot 2024-08-14 at 09 17 45

**Preview: `fill_in_details_form.html`**
Screenshot 2024-08-15 at 12 25 18

**Preview for cobrands that are overriding `_report_banner.html`**
Commit: [6c8739d](https://github.com/mysociety/fixmystreet/pull/5109/commits/6c8739dc8177bf222970130a97fdec0548e54fa1)

After we happy with the PR, I'll squash the commits. Currently is split into two, in case we decide we don't want to touch the cobrands that have overrides.

Screenshot 2024-08-15 at 12 29 11
Screenshot 2024-08-15 at 12 30 29
Screenshot 2024-08-15 at 12 31 18
Screenshot 2024-08-15 at 12 32 11
Screenshot 2024-08-15 at 12 32 56
Screenshot 2024-08-15 at 12 33 43

### CODE REVIEW

**Comment by lucascumsille** [comment]
@bekkileaver I wanted to check with you first if you are happy with the copy.

**Comment by bekkileaver** [comment]
Copy looks fine to me. Thanks @lucascumsille

**Comment by lucascumsille** [comment]
Thanks @bekkileaver I'll use it then

File: templates/web/base/around/_report_banner.html [hunk 1]
```diff
@@ -2,5 +2,5 @@ <h1 class="big-green-banner">
     [% loc( 'Click map to report a problem' ) %]
 </h1>
 <p class="map-alternatives">
-    [% tprintf( loc('Can’t use the map? <a href="%s" rel="nofollow">Skip this step.</a>'), url_skip) %]
+    [% tprintf( loc('Can’t use the map to start a report? <a href="%s" rel="nofollow">Skip this step →</a>'), url_skip) %]
```
Comment (davea) [inline templates/web/base/around/_report_banner.html:5]: Should this be encoded as `&rarr;`? Genuine question, I don't know the answer :) 

Also, does this have implications for RTL support? i.e. in a right-to-left localisation should this point left?

**Review by davea — COMMENTED** [review]
A couple of questions about this. On the face of it, this is fine though!

File: templates/web/base/around/_report_banner.html (hunk 1 above)
Comment (dracos) [reply templates/web/base/around/_report_banner.html:5]: Could we possibly keep the same text in both situations to keep one translation? Unless we think it makes a big difference, would `Can't use the map to start a report? <a href="%s" rel="nofollow">Skip this step</a>` (no arrow, no full stop) be okay in both places? That deals with the RTL issue (which I think is, yes, the arrow in the green banner swaps round automatically, for example).

Comment (lucascumsille) [reply templates/web/base/around/_report_banner.html]: I think that should work as well, it won't have much difference. I added a fixup for it.
Let me know if that is ok

File: templates/web/base/report/new/fill_in_details_form.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
 %]
     <ul class="change_location">
         <li class="change_location__map">[% loc('Click the map or drag the pin to adjust the location') %]</li>
-        <li class="screen-reader-only skip-the-map">[% tprintf( loc('Can’t use the map? <a href="%s" rel="nofollow">Skip this step.</a>'), url_skip) %]</li>
+        <li class="screen-reader-only skip-the-map">[% tprintf( loc('Can’t use the map to start a report? <a href="%s" rel="nofollow">Skip this step.</a>'), url_skip) %]</li>
```
Comment (dracos) [inline templates/web/base/report/new/fill_in_details_form.html:17]: Missed one full stop here

**Review by dracos — CHANGES_REQUESTED** [review]
One full stop is all (plus I think this has merged in some changes from a different PR about the footer nav?)

**Comment by lucascumsille** [comment]
Thanks @dracos I got rid of all the full stops and the odd extra commits that I added.

**Review by dracos — APPROVED** [review]
Thanks :)

**Pull request merged**